lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -41,7 +41,14 @@ use virt_utils;
 
 our @EXPORT
   = qw(download_network_cfg prepare_network restore_standalone destroy_standalone restart_network restore_guests restore_network
-  destroy_vir_network restore_libvirt_default enable_libvirt_log upload_debug_log check_guest_status check_guest_module save_guest_ip test_network_interface hosts_backup hosts_restore);
+  destroy_vir_network restore_libvirt_default enable_libvirt_log upload_debug_log check_guest_status check_guest_module save_guest_ip test_network_interface hosts_backup hosts_restore get_pool_available_space);
+
+sub get_pool_available_space {
+    if (is_xen_host()) {
```

Comment:
```bash
# virsh pool-info default | grep ^Available | awk '{print $2}'
183.65
```
This works on KVM too.

Reply:
Good point! should be coved this both on KVM and XEN together. 
Support to ensure available disk space for default(images) pool both kvm and xen together.